gradle.properties

```diff
@@ -25,10 +25,11 @@ jei_version=17.3.0.49
 graphlib_version=3.0.4
 graphlib_version_range=[3.0.4,)
 ae2_version=17.6.2-beta
+athena_version=EwkHR2YR
 #top_version=5.0.0
 top_version=1.20.0-10.0.0
 patchouli_version=1.19.3-78
-mekanism_cf_id=3922056
```

Comment:
I would use the mekanism maven, see my Chemical conduits PR.

Reply:
Good call, I'll change it to use the mek maven :)